README.md

```diff
@@ -144,6 +144,10 @@ const login = useGoogleLogin({
   Sign in with Google 🚀{' '}
 </MyCustomButton>;
 ```
+To access the user data fetch
+```
+https://www.googleapis.com/oauth2/v3/userinfo?access_token=<the access token from the useGoogleLogin() tokenResponse>
```

Comment:
the [discovery document](https://developers.google.com/identity/openid-connect/openid-connect#discovery) suggests that we should use https://openidconnect.googleapis.com/v1/userinfo instead